packages/common/src/tools/LogThrottler.ts

```diff
@@ -0,0 +1,81 @@
+import { Logger, LogLevel } from './Logger'
+
+interface LogInfo {
+  count: number
+  throttleCount: number
+  isThrottling: boolean
+}
+
+export interface LogThrottlerOptions {
+  threshold: number
+  thresholdTime: number
+  throttleTime: number
+}
+
+export class LogThrottler {
```

Comment:
This class could use a two sentence comment. Why we are throttling and what exactly these options do.